### PR TITLE
Update android configurations

### PR DESCRIPTION
**Pull request by deepak786**

### CODE REVIEW

File: intercom_flutter/android/build.gradle [hunk 1]
```diff
@@ -9,7 +9,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.1'
+        classpath 'com.android.tools.build:gradle:4.2.2'
```
Comment (Zazo032) [inline intercom_flutter/android/build.gradle]: Latest is 7.0.4

File: intercom_flutter/android/gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-all.zip
```
Comment (Zazo032) [inline intercom_flutter/android/gradle/wrapper/gradle-wrapper.properties]: Latest is 7.3.3

File: intercom_flutter/example/android/app/build.gradle [hunk 3]
```diff
@@ -39,7 +39,7 @@ android {
     defaultConfig {
         applicationId "io.maido.intercomexample"
         minSdkVersion 21
-        targetSdkVersion 29
+        targetSdkVersion 30
```
Comment (Zazo032) [inline intercom_flutter/example/android/app/build.gradle]: Latest is 31

**Review by Zazo032 — APPROVED** [review]
Can you try if using the latest versions available works?

File: intercom_flutter/android/build.gradle (hunk 1 above)
Comment (deepak786) [reply intercom_flutter/android/build.gradle]: should we update to 7.0.4? I think the Gradle plugin needs to be updated to the latest as well.

**Pull request merged**